core/src/main/scala/org/genivi/sota/core/rvi/SotaServices.scala

```diff
@@ -105,7 +105,10 @@ class SotaServices(updateController: ActorRef, resolverClient: ExternalResolverC
 
   def updatePackagesInResolver( message: InstalledPackages ) : Future[Unit] = {
     log.debug( s"InstalledPackages from rvi: $message" )
-    resolverClient.setInstalledPackages(message.device, message.installed_software)
+    for {
```

Comment:
Should I also change the non-rvi invocation to also rely on resolver to pass packages to device reg?